test/Fuzzing/fuzzer_test.swift

```diff
@@ -8,21 +8,21 @@
 // XFAIL: OS=watchos
 // CHECK: Crash!
 
-#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
-  import Darwin
-#elseif os(Linux) || os(FreeBSD) || os(OpenBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
-  import Glibc
-#elseif os(Windows)
-  import MSVCRT
-#else
-#error("Unsupported platform")
+#if canImport(Darwin)
+import Darwin.C
+#elseif canImport(Glibc)
+import Glibc
+#elseif canImport(MSVCRT)
+import MSVCRT
 #endif
 
-@_cdecl("LLVMFuzzerTestOneInput") public func fuzzOneInput(Data: UnsafePointer<CChar>, Size: CLong) -> CInt {
-  if (Size >= 3 && Data[0] == 65 && Data[1] == 66 && Data[2] == 67) {
-    fputs("Crash!", stdout);
-    fflush(stdout);
-    exit(1);
+@_cdecl("LLVMFuzzerTestOneInput")
+public func test(_ start: UnsafeRawPointer, _ count: Int) -> CInt {
+  let bytes = UnsafeRawBufferPointer(start: start, count: count)
+  if bytes.starts(with: "ABC".utf8) {
+    fputs("Crash!", stdout)
```

Comment:
I suppose this could just be `print` now...

Reply:
I did try `print` earlier, and it works, but the `fflush` is still required. I thought [`exit`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/exit.html) is supposed to "flush all open streams"?

I also tried `fatalError`, and redirecting stderr to FileCheck, so that Darwin/Glibc/MSVCRT isn't required. But I couldn't get that to work.